### DISCUSSION TITLE
Plasticity Parameters

### DISCUSSION THREAD

**Discussion by mehbah** [neperfepx/FEPX#11] [Q&A] [answered]
Hi There,
I'm going to use NEPER and FEPX to study the crystal plasticity behaviour of the AlSi10Mg alloy with different Si morphology and distribution. I have constructed a polycrystal with two phases of AlMg(or Al) and Si using NEPER. I need to define the Si phase as a hard phase or reinforcement phase and set the appropriate plasticity parameters in the materials.config file. Could you please help me to set the parameters or suggest other ways to do that?
Kind regards,
Mehran

**Comment by mkasemer** [comment, marked as answer]
Hello @mehbah,

I am not an expert on the specific polycrystalline system you are studying, and I cannot give good advice as to what parameters you should use. Finding a good set of parameters is often a challenge, and some results are sensitive to changes in the parameters used, so you want to have the highest confidence possible in the parameters used.

Generally, you will either have to measure your own parameters (experimental tests), or consult literature to determine what parameters are best representative of your material. Since you are able to identify the individual phases of your material, I would suggest finding studies that focus on each phase - there are many studies published on the use of crystal plasticity for both aluminum alloys and silicon. These studies will give you a sense of the average values and ranges for the plasticity parameters. You will likely find that there is a wide variety of values reported for each phase, and you will have to judge which values you trust the most based on the rigor of the respective study.

Best of luck!

**Reply by mehbah** [reply]
Okay! many thanks for your help!
